### ISSUE TITLE
\HeadlessChromium\Input\Mouse::find - Breaks future find + click

### ISSUE THREAD

**Issue by laurinkeithdavis** [chrome-php/headless-chromium-php#272]
This works:
```php
		try{

			$oMouse = ChromePHP::getPage()->mouse();
			$oMouse->find(
				"#EmployeeContactList")->click();
			ChromePHP::getPage()->waitForReload(
				ChromePHP::getPage()::LOAD,
				5000);
			ChromePHP::takeScreenshotAndWriteSourceToFile();

		}
		catch(Exception $oException){
```

But this does not work - click does not load new page, I get an operation timed out exception:

```php
		try{

			$oMouse = ChromePHP::getPage()->mouse();

			$oMouse->find(
				".highlight");
			$oMouse->find(
				"#EmployeeContactList")->click();
			ChromePHP::getPage()->waitForReload(
				ChromePHP::getPage()::LOAD,
				5000);
			ChromePHP::takeScreenshotAndWriteSourceToFile();

		}
		catch(Exception $oException){
```

I tried scrolling the to bring the $EmploeeContantList back in focus, but still fails.
```php
			$oMouse->scrollUp(
				10000);
```
However, if I don't do the click and just grab a screenshot, the scrolling does work, but the click still fails.

**Comment by enricodias** [comment]
The method `scrollToBoundary()` used by `find()` doesn't support scrolling up and left yet. In your example, if ".highlight" scrolls bellow the position of "#EmployeeContactList", it won't be able to scroll back to find it.

Scrolling all the way up with `scrollUp()` should _probably_ work. Can you provide a minimal reproducible example so I can debug it here? Including the html.

**Comment by laurinkeithdavis** [comment]
I will try, but that may take more time than I have any time in the near future.

**Comment by laurinkeithdavis** [comment]
I've tried and I would have to extricate too much to produce a standalone example that I can't justify the time to do so, we'll just have to split the test into multiple tests and load the page twice.

**Comment by laurinkeithdavis** [comment]
Wow, even if I split up the tests and reload the page it fails (will not click the link). Seems that once I use the mouse to find an element, I can't use it again.

**Comment by enricodias** [comment]
The unit tests we have do reload pages and use the find more than one time. Without a reproducible example it would be hard to guess what's happening.

**Comment by laurinkeithdavis** [comment]
https://github.com/chrome-php/chrome/issues/292

**Comment by upmanager** [comment]
I saw the Dom feature on version 1.2. It's working well.
I hope to make the release of version 1.2 soon.
https://github.com/chrome-php/chrome/tree/1.2

**Comment by enricodias** [comment]
Fixed in #320
